Resources/Catalog/FileCatalogClient.py

```diff
@@ -159,23 +159,23 @@ def getDirectoryMetadata( self, lfns, rpc='', url='', timeout=120):
         result['Value']['Successful'][path]['OwnerDN'] = res['Value'][0]
       else:
         result['Value']['Successful'][path]['OwnerDN'] = ''
-      result['Value']['Successful'][path]['OwnerDRole'] = getVOMSAttributeForGroup( group )
+      result['Value']['Successful'][path]['OwnerRole'] = getVOMSAttributeForGroup( group )
```

Comment:
This is the only change in the code... Sorry as all the rest is formatting automatically done by Eclipse